onnx/defs/generator/defs.cc

```diff
@@ -50,8 +49,7 @@ TensorProto message.
     .Attr(
           "shape",
           "The shape of the output tensor.",
-          AttributeProto::INTS,
-          OPTIONAL)
```

Comment:
Could there be cases that the shape of the output is dynamic?

Reply:
if it's dynamic, we should use shape as the input, see RandomUniformLike.